### PR TITLE
implementing some chips functionality

### PR DESCRIPTION
**Pull request by mansona**
Demo: https://cloudup.com/ckGCOdfqX2S

I've also added custom templates which is not in the gif above ^^

### CODE REVIEW

**Comment by mansona** [comment]
I've updated this so it works on new master

File: tests/dummy/app/templates/chips.hbs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    {{/paper-card}}
+
+    <h3>Template</h3>
+    {{#code-block language='handlebars' }}
+&lt;h2&gt;Default Template&lt;/h2&gt;
+\{{paper-chips content=fruitNames readOnly=readOnly removeItem=&quot;removeItem&quot; addItem=&quot;addItem&quot; placeholder=&quot;Add a tag&quot;}}
+
+&lt;h2&gt;Custom Template&lt;/h2&gt;
+\{{#paper-chips readOnly=readOnly removeItem=&quot;removeCustomItem&quot; addItem=&quot;addCustomItem&quot; placeholder=&quot;Add a tag&quot; content=customFruitNames as |item|}}
+  &lt;strong&gt;{{item}}&lt;/strong&gt;
```
Comment (miguelcobain) [inline tests/dummy/app/templates/chips.hbs]: We need to escape `{{item}}`.

**Comment by miguelcobain** [comment]
Can we include contact chips in this PR?
I'm not opposed to adding them later, but I think they're easy to accomplish from there. Right?

**Comment by mansona** [comment]
Yea I could probably throw it in there sometime soon, the thing that got me stumped was that I wanted to implement the same functionality as this page: https://material.angularjs.org/latest/#/demo/material.components.chips but we don't have autocomplete implemented yet. I suppose we could implement it without autocomplete :joy:

**Comment by peec** [comment]
@mansona When #132 is merged you got `paper-autocomplete` to use.

File: tests/dummy/app/styles/app.scss [hunk 2]
```diff
@@ -6,7 +6,6 @@
  */
 
 body {
-  overflow: hidden;
```
Comment (mansona) [inline tests/dummy/app/styles/app.scss]: I don't know why I needed to do this but if I didn't remove this I wasn't able to scroll the dummy app

**Comment by mansona** [comment]
Ok so I've brought this up to the latest and have even done a quick contact-cards implementation :tada: If you're not happy with the contact-cards implementation I would suggest taking you up on your offer @miguelcobain to add them later i.e. remove them and worry about them later. 

I don't know why this is failing the build, can't figure it out from the output of travis. Can someone help me with this?

**Comment by miguelcobain** [comment]
@mansona Sorry for asking you this *after so much time*.

We're preparing release v1 which will be the big thing in ember-paper.
Please read #249.
We're already automatically importing angular material sass styles.

It would be great if you could bring this to be on par with latest v1 branch and angular material 1.
I have no idea if a lot has changed or not.

**Comment by mansona** [comment]
I'll see if I can take a look sometime over the next few days :+1:

**Comment by mansona** [comment]
Ok so now I'm going to close this in favour of https://github.com/miguelcobain/ember-paper/pull/267 I have quite a few issues with this new branch but I'll start the conversation there :+1:

**Pull request closed without merging**